src/randomizer.py

```diff
@@ -1126,6 +1126,31 @@ def perm_reorder_stmts(
     ast_util.insert_statement(tob, toi, stmt)
 
 
+def perm_equals_operator(
+    fn: ca.FuncDef, ast: ca.FileAST, indices: Indices, region: Region, random: Random
+) -> None:
+    """Convert a statement of the form `x = x op y` to `x op= y`"""
+    cands: List[ca.Assignment] = []
+    operators = ["+", "-", "*", "/", "<<", ">>", "^", "|", "&"]
+
+    class Visitor(ca.NodeVisitor):
+        def visit_Assignment(self, node: ca.Assignment) -> None:
+            if node.op == "=" and \
+            region.contains_node(node) and \
+            isinstance(node.rvalue, ca.BinaryOp) and \
+            ast_util.equal_ast(node.lvalue, node.rvalue.left) and \
+            node.rvalue.op in operators:
```

Comment:
```suggestion
            if (node.op == "=" and
            region.contains_node(node) and
            isinstance(node.rvalue, ca.BinaryOp) and
            ast_util.equal_ast(node.lvalue, node.rvalue.left) and
            node.rvalue.op in operators):
```
and then let `black` auto-format this